src/main.rs

```diff
@@ -178,7 +178,7 @@ fn main() -> crossterm::Result<()> {
 
     if opt.list_languages {
         opt.languages()
-            .expect("Couldn't get installed languages under config directory. Make sure the config directory exists.")
+            .expect("Couldn't get installed languages under the config directory. Make sure the config and language directory exist.")
```

Comment:
This improves the error message.
However I agree with @omarkohl, the message should be printed as a warning and return an empty list with `.or_else(vec![])`.

Reply:
Since there's no logging this could look like the following:
```rust
    if opt.list_languages {
        opt.languages()
            .or_else({
                println!("Warning: Couldn't get installed languages under config directory. Make sure the config directory exists.");
                vec![]
            })
            .iter()
            .for_each(|name| println!("{}", name.to_str().expect("Ill-formatted language name.")));
        return Ok(());
    }
```